src/wallet.py

```diff
@@ -0,0 +1,92 @@
+import json
+from json.decoder import JSONDecodeError
+
+import structlog
+from nacl.encoding import HexEncoder
+from nacl.signing import SigningKey
+import base64
+from cryptography.fernet import Fernet
+
+
+logger = structlog.get_logger(__name__)
+
+class wallet:
+    PRIVATE_KEY = ""
+    PUBLIC_KEY = ""
+    
+    def generate_wallet():
+        password = input("\nSe necesita una contraseña, escriba una: ")    
```

Comment:
Can we provide the `password` as an input to `generate_wallet()`? I would prefer to not have blocking functions like `input()` inside here.

Reply:
Included